### DISCUSSION TITLE
In DevConfig keine Übertragung von Einstellungen an TRV möglich.

### DISCUSSION THREAD

**Discussion by MaxR8684** [OpenCCU/OpenCCU#2941] [🇩🇪 Fragen&Antworten] [answered]
Hi zusammen, ich versuche den TEMPERATUREFALL_MODUS auszuschalten, habe dazu die DevConfig aktiviert. Leider kommt im Browser jedes Mal eine Meldung, dass "die Seite nicht erreichbar ist". 
Kenn mir jmd helfen, ob das bekannt ist und was ich ggf falsch mache?

**Comment by Baxxy13** [comment]
Welches "TRV" genau? Es gibt zig verschiedene.
Wenn es kein eTRV-C ist dann sollte das über die Geräteeinstellungen (de)aktivierbar sein mit RM 3.79.6.20241122

**Reply by MaxR8684** [reply]
Hi und danke für die schnelle Antwort, ich habe mehrere HmIP-eTRV-2 I9F und ein HmIP-eTRV-E. 

Ich hab RM 3.79.6.20241122 als Addon von Home Assistant laufen. Ich kann das leider in den Geräteeinstellungen nicht finden.

Kannst du mir sagen wo genau das sein müsste?

VG Max

**Reply by Baxxy13** [reply, marked as answer]
Modus für Temperatursturzerkennung, siehe:
https://github.com/jens-maus/RaspberryMatic/issues/2893#issuecomment-2481051754

**Reply by MaxR8684** [reply]
Vielen Dank für diesen Hinweis, ich habe entdeckt, dass ich im einfachen Modus unterwegs war.
Nach dem umschalten auf den Expertenmodus sind die gewünschten Einstellungen da.

**Reply by jens-maus** [reply]
> Vielen Dank für diesen Hinweis, ich habe entdeckt, dass ich im einfachen Modus unterwegs war.
> Nach dem umschalten auf den Expertenmodus sind die gewünschten Einstellungen da.

Ich sag ja, dieser doofe "Einfache Modus" muss weg. Man schießt sich da nur regelmäßig damit ins Knie.
